src/main/kotlin/org/wycliffeassociates/otter/common/domain/content/FileNamer.kt

```diff
@@ -40,7 +40,7 @@ class FileNamer(
             bookSlug,
             "c${formatChapterNumber()}",
             formatVerseNumber()?.let { "v$it" },
-            sort?.let { "s$it" },
+            formatSort(),
```

Comment:
you have three different ways of formatting this:

"c${formatChapterNumber()}"
formatVerseNumber()?.let { "v$it" }
formatSort()

seems to me that only the last one is done "correctly" as the first two imply that the result of the "format" function call is still not "formatted."